lib/webdriveragent.js

```diff
@@ -399,6 +403,14 @@ class WebDriverAgent {
   set url (_url) {
     this._url = url.parse(_url);
   }
+
+  get fullyStarted () {
```

Comment:
is there a compelling reason to have two different names for the same thing here? could we pick whichever one is more informative, and simply apply a comment in non-obvious places? not totally opposed to this though.